osu.Game/Overlays/BeatmapSetOverlay.cs

```diff
@@ -124,7 +127,7 @@ protected override void PopIn()
         protected override void PopOut()
         {
             base.PopOut();
-            header.Details.StopPreview();
+            previewTrackManager.CurrentTrack?.Stop();
```

Comment:
this seems dangerous. you're stopping a global track on closing this overlay, but what if that track was actually started by an element that is still being displayed?

Reply:
Looks like my mistake. This can be resolved by adding `PreviewTrack.Owner` property of type `OverlayContainer`.